src/status.ts

```diff
@@ -138,12 +149,14 @@ abstract class Button {
 }
 
 class WorkspaceButton extends Button {
-  private static readonly _autoSelectToolTip = localize('active.folder.auto.select.tooltip', 'Active folder');
-  private static readonly _toolTip = localize('active.folder.tooltip', 'Select Active folder');
+  // private static readonly _autoSelectToolTip = localize('active.folder.auto.select.tooltip', 'Active folder');
+  // private static readonly _toolTip = localize('active.folder.tooltip', 'Select Active folder');
+  private static readonly _autoSelectToolTip = localize('active.folder.auto.tooltip', 'auto');
 
   settingsName = 'workspace';
   command = 'cmake.selectActiveFolder';
   icon = 'folder-active';
+  tooltip = localize('click.to.select.workspace.tooltip', 'Click to select the active workspace');
```

Comment:
It should say active "folder", not "workspace" to avoid confusing the terminology with what VS Code uses.

Reply:
fixed.